apps/meteor/app/federation/server/endpoints/dispatch.js

```diff
@@ -134,15 +139,24 @@ const eventHandlers = {
 				if (persistedSubscription) {
 					// Update the federation, if its not already set (if it's set, this is likely an event being reprocessed
 					if (!persistedSubscription.federation) {
-						await Subscriptions.updateOne({ _id: persistedSubscription._id }, { $set: { federation: subscription.federation } });
+						const { modifiedCount } = await Subscriptions.updateOne(
+							{ _id: persistedSubscription._id },
+							{ $set: { federation: subscription.federation } },
```

Comment:
`federation` field is not on the list of `subscriptionFields`, so we don't need to notify when changing it.